README.md

```diff
@@ -1,2 +1,20 @@
 # Geomapfish Locator Plugin
 A QGIS 3 locator plugin for Geomapfish services
+
+## Quick start Guide
+
+Go to Extension -> Install and search for Geomapfish Locator
+
+Install the plugin
+
+Go to the locator options by clicking on the magnifier on the bottom left of the screen
+
+Set filter parameters for your search service :
+
+* Name of the filter (e.g. MysearchService)
+
+* URL (e.g. https://www.sigip.ch/search)
+
+* CRS (e.g. EPSG 21781)
+
+Click OK and search
```

Comment:
```suggestion
Validate and start search in the locator bar at the bottom left of the application.

If you want to avoid typing the prefix of the filter (`gmf`), enable it by default in the locator bar settings.
```